packages/attack-http-requests/src/index.ts

```diff
@@ -0,0 +1,47 @@
+import Mitm from 'mitm';
+
+export interface HTTPAttackOptions {
+  ignoreURLpatterns?: string[];
```

Comment:
Capitalisation here is confusing... I know `Url` is also annoying but stricter camelcase is better:

```suggestion
  ignoreUrlPatterns?: string[];
```

Reply:
Sure thing, updated.